modules/table/src/integrationTest/java/org/apache/ignite/internal/disaster/ItDisasterRecoveryReconfigurationTest.java

```diff
@@ -1234,6 +1242,295 @@ void testTwoPhaseResetOnEmptyNodes() throws Exception {
         assertPlannedAssignments(node0, partId, assignments13);
     }
 
+    @Test
```

Comment:
Could you please run given and other tests in the class with @RepeatedTest locally in order to verify that they are stable?

Reply:
I've checked all new tests. They seems rather stable. All failures that I've met were related to old issues.